mg400_interface/src/commander/dashboard_commander.cpp

```diff
@@ -94,13 +94,13 @@ uint64_t DashboardCommander::robotMode() const
   return static_cast<uint64_t>(ResponseParser::takeInt(response.ret_val));
 }
 
-void DashboardCommander::payload(
+void DashboardCommander::pay_load(
```

Comment:
The name roles for the method is camel-case in our project so it is better to change it `payLoad`

Reply:
Thank you for pointing it out. I'll fix it.